src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -46,22 +41,28 @@ public class EditCommand extends Command {
 
     public static final String ERROR_MESSAGE_NO_SUCH_GROUP = "Group does not exist in ProfBook.";
 
+    public static final String MESSAGE_USAGE =
+            "Usage: " + COMMAND_WORD + " <path> " + "[OPTION]... \n"
+            + "\n"
+            + "Edit the details of the group or student.\n"
+            + "\n"
+            + "Argument: \n"
+            + "    path                 Valid path to group or student\n"
+            + "\n"
+            + "Option: (Pass in at least one of the fields below to edit)\n"
+            + "    -n, --name           Name of the group / student\n"
+            + "    -i, --id             Id of the group / student\n"
+            + "    -e, --email          Email of the student\n"
+            + "    -p, --phone          Phone of the student\n"
+            + "    -a, --address        Address of the student\n"
```

Comment:
The description sounds weird, consider changing it to "Provide at least one of the following fields for editing."